wps_jobs/wps_jobs/weather_model_jobs/ecmwf_prediction_processor.py

```diff
@@ -77,8 +77,9 @@ def _should_interpolate(self, prev_prediction: ModelRunPrediction, prediction: M
             # Timestamps on the same day but surround 20:00 UTC should be interpolated
             return prev_timestamp.hour <= 18 and next_timestamp.hour >= 21
```

Comment:
We might be able to make this less error prone with:
`return prev_timestamp.hour < 20 and next_timestamp.hour > 20`

Actually, if we do this we wouldn't need the `if next_timestamp.hour == 20` I mentioned in the other comment.